Updating the Python SDK docs (#299)
up42/up42-py: docs/30-second-example.md

[![Binder](assets/badge_logo.svg)](https://mybinder.org/v2/gh/up42/up42-py/master?filepath=examples%2Fguides%2F30-seconds-example.ipynb)

### **Authentication**

The initial steps for [installing](installation.md) and [authentication](authentication.md) were covered in the previous sections. After, we need to create a project to set up and configuring the workflow of the application.

```python
import up42
up42.authenticate(project_id="project ID string", project_api_key="project-API-key") # inline authentication (also possible with json file)
project = up42.initialize_project()
```
### **Creating a workflow**

Creating a workflow only consists of adding a workflow name, and adding some input tasks. The lists for all the input tasks can be found in the documentation [**processing blocks section**](https://up42.com/marketplace/blocks/processing). 

```python
# Construct workflow
workflow = project.create_workflow(name="30-seconds-workflow", use_existing=True)
#print(up42.get_blocks(basic=True))
input_tasks = ["Sentinel-2 L2A Visual (GeoTIFF)",
               "Sharpening Filter"]
workflow.add_workflow_tasks(input_tasks)
```

### **Defining the input parameters**

> Input tasks are not enough to completely define our workflow. Also, input parameters should be defined which include the AOI (area of interest), dates of analysis, among others. The AOI can be added into the workflow by different methods, external geometry files, and operations as well as some example AOI included in our package. 

Reviewer: ```suggestion
To further configure the workflow, we provide workflow input parameters, e.g. the AOI (area of interest), dates of analysis, among others. The AOI geometry can be read via various methods (e.g. `up42.draw_aoi()`, `up42.read_vector_file()`) and provided as multiple formats (e.g. GeoJSON, shapely Polygon, GeoDataFrame).
```

                                                 end_date="2020-12-31",
                                                 limit=1)
input_parameters["esa-s2-l2a-gtiff-visual:1"].update({"max_cloud_cover":5})
```

> 

Reviewer: ```suggestion
### **Price Estimation and Pre-testing**
```

Price estimation is an important feature that allows to estimate the actual costs of your workflow before you run your application. Estimating costs are reported in credits units that you can check on your account dashboard.

```python
# Price estimation
workflow.estimate_job(input_parameters)
```

> ### **Pre-testing**

Reviewer: ```suggestion
```

Prior a final run of our workflow we can test the configuration and availability of the tasks included in our pipeline. This is done by running test jobs before the final run.

```python
# Run a test job to query data availability and check the configuration.
test_job = workflow.test_job(input_parameters, track_status=True)
```

> ### **Final run**

Reviewer: ```suggestion
### **Running the Workflow**
```
